pkg/registry/service/rest.go

```diff
@@ -342,6 +443,11 @@ func (rs *REST) Update(ctx api.Context, name string, objInfo rest.UpdatedObjectI
 		service.Status.LoadBalancer = api.LoadBalancerStatus{}
 	}
 
+	success, err := rs.healthCheckNodePortUpdate(oldService, service)
```

Comment:
Good point - yes, I need to add feature gates to the new calls in Delete and Update.